tests/test-portals.c

```diff
@@ -444,7 +444,7 @@ DEFINE_TEST_EXISTS(inhibit, INHIBIT, 3)
 DEFINE_TEST_EXISTS(location, LOCATION, 1)
 DEFINE_TEST_EXISTS(network_monitor, NETWORK_MONITOR, 3)
 DEFINE_TEST_EXISTS(notification, NOTIFICATION, 1)
-DEFINE_TEST_EXISTS(open_uri, OPEN_URI, 3)
+DEFINE_TEST_EXISTS(open_uri, OPEN_URI, 4)
```

Comment:
This feels like saying we have test coverage for OpenURI in version 4, but we don't. Would it be possible to add a test for this? Since it's just a simple call, it probably should be fairly trivial.

Reply:
Currently I'm not completely sure what to test. Maybe someone could direct me to the similar test?